web/locales/de.json

```diff
@@ -6,6 +6,29 @@
         "datasources": "Datenquellen",
         "contribute": "Wirke mit, indem du <a href=\"%s\" target=\"_blank\">dein Land hinzufügst</a>"
     },
+    "mobile-info-tab": {
+      "map-info-text": "Diese Karte zeigt in Echtzeit wie viel CO2 ausgestoßen wird. Wir färben alle Gegenden der Welt mit ihrer Kohlenstoffintensität (einschließlich aller Treibhausgase) der dort verbrauchten Elektrizität. Grünere Farben bedeuten Klima freundlichere Elektrizität.",
+      "electricity-origins-text": "Die Pfeile auf der Karte zeigen die Ströme der Elektrizität zwischen verschiedenen Gegenden. Klicke auf eine Gegend um mehr Informationen über die Herkunft der Elektrizität zusehen.",
+      "wind-solar-text": "Wind- und Solarenergie sind abhängig vom Wetter. Klicke auf den Sonnen- und Windknopf um die Echtzeit Stärke des Windes und des Sonnenscheins überall in der Welt anzuzeigen."
```

Comment:
The current German for these is:

        "map-info-text": "Diese Karte veranschaulicht in Echtzeit, wie viel CO2-Äquivalente bei der Stromerzeugung freigesetzt werden. Die Färbung der Zonen erfolgt anhand des spezifischen Ausstoßes von Treibhausgasen (CO2-Intensität), die durch die Deckung des Strombedarfs einer Region emittiert werden. Eine grüne Färbung steht dabei für eine klimafreundlichere Erzeugung.",
        "electricity-origins-text": "Die Pfeile auf der Karte stellen den physischen Stromaustausch zwischen Regionen dar. Tippe auf eine Region, um mehr Details zur Herkunft des verfügbaren Stromes zu erhalten.",
        "wind-solar-text": "Die Verfügbarkeit von Wind- und Solarstrom ist wetterabhängig. Tippe auf den Wind- und Sonnenbutton, um das globale Echtzeit-Potenzial dieser Ressourcen anzuzeigen."